src/framework/mlt.vers

```diff
@@ -552,3 +552,26 @@ MLT_6.20.0 {
     mlt_profile_scale_height;
     mlt_properties_set_string;
 } MLT_6.18.0;
+
+MLT_6.21.0 {
```

Comment:
I only use upcoming version in `mlt.vers` so that this does not need change upon release in a way that conflicts with a development version. For example, here `MLT_6.21.0` is based on `MLT_6.20.0` but then come release `MLT_6.22.0` would be based on `MLT_6.20.0` since there cannot be an empty version `MLT_6.22.0` based on `MLT_6.21.0` if you understand. Simply put, change `MLT_6.21.0` to `MLT_6.22.0` now.